scss/bootstrap/components/_date-picker.scss

```diff
@@ -0,0 +1,41 @@
+.mobile {
+  date-picker {
+    .chr_uib-date-picker {
+      display: none
+    }
+
+    .chr_html5-date-picker {
+      position: relative;
+
+      > input {
+        -webkit-appearance: none;
+        line-height: normal !important;
+
+        &::-webkit-inner-spin-button {
+          -webkit-appearance: none;
+          display: none;
+        }
+      }
+
+      span {
+        background-color: #FFF;
+        border-left: 1px solid #949CAA !important;
```

Comment:
I think this color is `$crm-color-neutral` ([source](https://github.com/civicrm/org.civicrm.shoreditch/blob/44a8c1006dd4e7caeb9e5cab08e1885e23b6b33b/scss/bootstrap/overrides/_variables.scss#L56))